src/EnergyPlus/WindowManager.cc

```diff
@@ -7243,8 +7341,8 @@ namespace WindowManager {
         Real64 temdiff; // Inside/outside air temperature difference (K)
         Real64 ressum;  // Resistance sum (m2-K/W)
 
-        rguess(1) = 1.0 / (state.dataWindowManager->hcout + hrad);
-        rguess(state.dataWindowManager->nglface + 1) = 1.0 / (hcinStartValue + hrad);
+        rguess(1) = 1.0 / (state.dataWindowManager->hcout + hrad) * state.dataHeatBal->CoeffAdjRatio(ConstrNum);
```

Comment:
Why not passing the ratio value?

Reply:
Changed to pass in adjustment ratio rather than ConstrNum